lmfit/lineshapes.py

```diff
@@ -145,6 +145,24 @@ def moffat(x, amplitude=1, center=0., sigma=1, beta=1.):
     return amplitude / (((x - center)/max(tiny, sigma))**2 + 1)**beta
 
 
+def pearson4(x, amplitude=1.0, center=0.0, sigma=1.0, expon=1.0, skew=0):
+    """Return a Pearson4 lineshape.
+
+    Using the Wikipedia definition:
+
+    pearson4(x, amplitude, center, sigma, expon, skew) =
+        amplitude*|gamma(expon + I skew/2)/gamma(m)|**2/(w*beta(expon-0.5, 0.5)) * (1+arg**2)**(-expon) * exp(-skew * arctan(arg))
```

Comment:
what is the "I" here (sorry I quickly Googled but didn't find the Wikipedia link immediately nor invested time in looking at the lineshape function)... it should be either a function argument or defined. Also I have a hard time relating the Python function you return to the definition of the Pearson4 lineshape you give, but again as I said I haven't spend much time at it...

Reply:
I still don't see a clear correlation between the function defined here and what is `return`-ed below as the lineshape function....  is that really only me ;)